consul/install.sls

```diff
@@ -47,6 +47,15 @@ consul-data-dir:
     - group: {{ consul.group }}
     - mode: 0750
```

Comment:
While this isn't part of this PRs changeset, there's an issue throughout this formula.

https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html:

> ##### Warning
> 
> When using a mode that includes a leading zero you must wrap the value in single quotes. If the value is not wrapped in quotes it will be read by YAML as an integer and evaluated as an octal.